src/components/TextField/index.tsx

```diff
@@ -203,6 +207,7 @@ const TextField: FunctionComponent<TextFieldProps> = forwardRef(
         <StyledInput
           hasError={hasError || !!errorMessage || false}
           isValid={typeof isValid !== 'undefined' ? isValid : !hasError && !errorMessage}
+          aria-required={required}
```

Comment:
we are already passing in "required" with {...props}
and HTML attributes are king, so if we are sending that we don't want to send aria-required as well. When we belt and braces standard mark that screen reader can read anyway AND the corresponding aria-label, we can end up with weird screen reader behaviour reading things out twice or simple breaking because they can be flakey as hell. 
We really should only use aria-label to fix things that we can't sort out with the appropriate markup.

Reply:
no worries- remove line